todo/src/components/TodoSubmitter.js

```diff
@@ -0,0 +1,43 @@
+//do I need this? Or just add the submitter form to List?
+import React from 'react';
+import { connect } from 'react-redux';
+import { addTodoAction} from '../actions'
+// src\actions\index.js
+class TodoSubmitter extends React.Component {
+    constructor() {
+        super()
+        this.state = {
+            text: '',
+            completed: false
```

Comment:
Why did you choose to set the default completed status here instead of the reducer?